apps/dotcom/zero-cache/migrations/024_add_fairy_state.sql

```diff
@@ -0,0 +1,5 @@
+-- Purpose: Add a fairyState column to the file_state table that stores per-user, per-file fairy agent state.
+
+ALTER TABLE file_state
+ADD COLUMN "fairyState" VARCHAR;
```

Comment:
@MitjaBezensek lemme know if this stuff looks ok to you, too.
it's used in `FairyApp.tsx`